src/deepsparse/utils/onnx.py

```diff
@@ -68,6 +68,49 @@
 }
 
 
+def save_onnx_to_path(model: Model, model_path: str, external_data_file: str) -> bool:
+    """
+    Save model to the given path.  If the model has external data, store the
+    external data in 'external_data_file'.
+    Returns False if the model had no external data, True otherwise.
+    """
+    try:
+        onnx.checker.check_model(model)
+        onnx.save(model, model_path)
+        return False
+    except ValueError:
+        # Presumably the checker has failed because of external data
+        onnx.save_model(
```

Comment:
do we not also need to call `convert_model_to_external_data` 

(`from onnx.external_data_helper import convert_model_to_external_data`)

Reply:
afaict that doesn't seem to be necessary.